woocommerce/woocommerce
# Local Pickup Settings Screen Fixes

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR fixes 3 issues relating to the local pickup settings screen (Settings > Shipping > Local Pickup)

Fixes #52400 There was an error when sorting items
Fixes #52356 There were some styling issues with the enable/disable toggle
Fixes #52357 Settings could not be saved after sorting locations



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to Settings > Shipping > Local Pickup
2. Add some locations and save changes
3. Ensure the "enabled" toggles on each location row are aligned right
4. Open your browser error console
5. Drag and drop to sort the locations
6. Confirm there are no errors/warnings logged as a result of this action, and that the "save" button is clickable



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
